### DISCUSSION TITLE
Primeiro Encontro

### DISCUSSION THREAD

**Discussion by antoniosousa** [clube-de-leitura-python/pense-em-python-2021#5] [Conversa geral]
O que houve no primeiro encontro, devido o horário infelizmente não consigo acompanhar, mas eu gostaria de ficar atualizado :)

**Comment by edmilsonlibanio** [comment]
Eu também gostaria de saber. Por conta do horário não tive como acessar. Seria interessante fosse possível ficar gravado..

**Comment by vitorestevam** [comment]
Rolaram uns problemas técnicos então o encontro foi curto. Foi falado um pouco sobre o código de conduta e sobre a motivação de algumas pessoas dps falamos sobre o autor.

A proposta, segundo o Alexandre, é que o grupo siga um ritmo que todos consigamos levar e que quem tiver mais adiantado possa ajudar os outros. 

Ficou combinado que pra dia 13 seriam os caps 1 e 2 do livro e que as discussões seriam fomentadas por aqui 😉

esse pad aqui foi usado no encontro, então tem uma síntese do que foi discutido [pad.education](https://pad.education/p/r.e0decf1a09206cabd1bfd3c531f080e8)

* [Fórum pra caso queira se apresentar](https://github.com/clube-de-leitura-python/pense-em-python-2021/discussions/6)
* [Informações sobre o grupo](https://github.com/clube-de-leitura-python/pense-em-python-2021)

**Reply by villares** [reply]
Boa, @VitorEstevam, obrigado pelo relato!  Foi bem isso!

Troquei o seu link do pad por uma versão somente leitura pra evitar robôs de spam :)

**Reply by sarahmrds** [reply]
na parte de "Pessoas / área de interesse / contato" vão estar só os adms ou é pra adicionar o pessoal que ta acompanhando esse projeto pra aprender?

**Reply by villares** [reply]
Oi @sarahmrds todo mundo que quiser pode por sim! De  "admin" hoje tinha só eu a Carolina e o Rodrigo.
Se você olhar no histórico da mensagem do VitorEstevan (_edited by villares_) dá pra pegar o link do pad editável...
![image](https://user-images.githubusercontent.com/3694604/127727259-14c4cbfc-8425-4a89-b9e0-9fa2dc6b064f.png)

**Comment by CatarinaLuciane** [comment]
Obrigada por colocarem o que rolou aqui! Hoje não pude, mas no próximo encontro farei de tudo pra estar presente.
Me adiantei e estou tentando resolver os exercícios do capítulo 2. 

Alguém sabe onde encontro as respostas? Tentei pesquisar mas não tive muito sucesso.

**Reply by villares** [reply]
Ótima pergunta @CatarinaLuciane, também não achei. Acho que podíamos fazer um thread aqui no fórum sobre isso com as respostas que encontrarmos/produzirmos.

**Reply by CatarinaLuciane** [reply]
Verdade, seria ideal! 
Eu tinha visto que na página do autor, em tese, teria as respostas do livro, mas fucei os códigos e nada lá parece com os exercícios...

Já tenho a resolução de 1 problema hahaha espero que esteja certo, tive de estudar 586568 coisas pra descobrir a resposta hahahaha

**Reply by villares** [reply]
Poxa, tenho essa vaga lembrança de ter visto o autor mencionar sobre respostas também, mas também não estou mais encontrado. Vamos sim registrar! 

Confesso que não curti o problema da corrida... mas vamos lá né :D
